api/controllers/v1/jobs.ts

```diff
@@ -137,32 +137,18 @@ async function NotifyBuildSummary(jobId: string): Promise<any> {
   };
 }
 
-function limit_message_size(message) {
-  while (message.length >= 256) {
-    let end = 255;
-    while (message[end] != ' ') {
-      end -= 1;
-    }
-    message = message.substring(0, end + 1);
-  }
-  return message;
-}
-
 function extractUrlFromMessage(fullDocument) {
```

Comment:
Should we add a couple quick unit tests for this function?

Reply:
sure

Reply:
added